components/atom/input/src/Input/Component/index.scss

```diff
@@ -52,7 +53,7 @@ $class-read-only: '#{$base-class}--readOnly';
     }
   }
 
-  @each $state, $color in $states-atom-input {
```

Comment:
IMHO since all form components use the variable: `$states-atom-input` that comes from this extend: `@extend %sui-atom-input-input;`, it is better to add the `alert` in the `$states-atom-input` variable instead of creating a new one (`$c-atom-input-type`), so the rest of components, for example: autosuggest, select, etc (that use the `$states-atom-input` variable) will also be able to have the `alert` if we add the corresponding tokens.

Reply:
otherwise if someday we change the states of the input in this variable `$states-atom-input` will not apply in this component. In other words, in this PR we are creating various versions ($states-atom-input and $c-atom-input-type) of the same 'truth' (the states of the input) taking into account that a lot of components use the input component.

Reply:
check for example here, `molecule/autosuggest` is using `$states-atom-input` variable: https://github.com/SUI-Components/sui-components/blob/329054791ae649ea0ae2bcbfc43860b9d34b4036/components/molecule/autosuggest/src/index.scss#L65

Reply:
Thanks for your help! I'm adding the `alert` state at `sui-theme` 👍